plugins/outputs/quix/quix.go

```diff
@@ -0,0 +1,130 @@
+// quix.go
+package quix
+
+import (
+	"crypto/sha256"
+	"strings"
+
+	"github.com/IBM/sarama"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/outputs"
+	"github.com/influxdata/telegraf/plugins/serializers"
+)
+
```

Comment:
You need to embed the sample.conf file...
```suggestion

//go:embed sample.conf
var sampleConfig string

```